app/src/androidTest/java/org/mozilla/focus/activity/NavigationTest.java

```diff
@@ -99,6 +101,9 @@ public void browsingWebsiteBackAndForward_backAndFrowardToWebsite() {
                 .check(matches(withText(TARGET_URL_SITE_2)));
         IdlingRegistry.getInstance().unregister(loadingIdlingResource);
 
+        // play around the web content view
+        onView(withId(R.id.main_content)).perform(swipeUp(), swipeDown());
```

Comment:
what we really want is to get the correct timing to perform "go back"
This works but not ideal and is misleading.
Maybe we can just add a comment and a TODO here.
Please help file a new issue to track this work.